node/test/OpenT2TTests.ts

```diff
@@ -141,12 +142,29 @@ test("Logger with default parameters can be instantiated multiple times", async
 });
 
 test("Logger can be instantiated with custom file parameter", async t => {
-    let logger = new Logger(undefined, testLogFileName) ;
+    let logger = new Logger(undefined, testLogFileName);
     logger.info("Writing default level to default console + file.");
     logger.warn("writing warn level to default console + file.");
     t.is(logger.getConfiguredTransports().length, 2);
 });
 
+test("Logger with default parameters can log custom loggingMetadata parameter like a correlationVector", async t => {
+    let logger = new Logger();
+    let stringObject = "Random String"
+    logger.info("Writing default level to default console with loggingMetadata:", stringObject, "ABC123");
+    logger.warn("writing warn level to default console with loggingMetadata:", undefined, "XYZ456");
```

Comment:
this would be not-user-friendly - might want to switch the parameters around or as Arjun suggested better still, make this a dictionary where the first is the metadata.